src/sage/rings/finite_rings/finite_field_constructor.py

```diff
@@ -637,11 +651,20 @@ def create_key_and_extra_args(self, order, name=None, modulus=None, names=None,
             Finite Field in aa of size 7^2
         """
         for key, val in kwds.items():
-            if key not in ['structure', 'implementation', 'prec', 'embedding', 'latex_names']:
+            if key not in ['structure', 'implementation', 'prec', 'embedding', 'latex_names', 'impl']:
```

Comment:
```suggestion
            if key not in ['structure', 'prec', 'embedding', 'latex_names', 'impl']:
```
Not needed since `implementation` is an explicit keyword argument.